scanner/src/test/kotlin/scanners/fossid/FossIdTest.kt

```diff
@@ -379,14 +379,14 @@ class FossIdTest : WordSpec({
                 .expectCreateScan(projectCode, scanCode, vcsInfo)
                 .expectDownload(scanCode)
                 .mockFiles(scanCode)
-            coEvery { service.runScan(USER, API_KEY, scanCode) } returns EntityResponseBody(status = 1)
+            coEvery { service.runScan(USER, API_KEY, scanCode, any(), any()) } returns EntityResponseBody(status = 1)
```

Comment:
Instead of `any()`, concrete values should be used, since the formatting options are known.